srtcore/core.cpp

```diff
@@ -5550,7 +5561,7 @@ void * srt::CUDT::tsbpd(void* param)
         if (self->m_bClosing)
             break;
 
-        SRT_ATR_UNUSED bool bWokeUpOnSignal = true;
+        bool bWokeUpOnSignal = true;
```

Comment:
The scope of this variable can be reduced. It is used both in the `if` and `else` statements, but independently. Thus can be defined inside those.

Reply:
You are right, this was likely tied to some earlier version that provided some log instruction after this if-branch, hence the scope was bigger. Now it has its own log in every branch, so this isn't necessary.